core/src/main/kotlin/com/malinskiy/marathon/Marathon.kt

```diff
@@ -181,8 +181,6 @@ class Marathon(
     }
 
     private fun logSystemInformation() {
-        log.info { "System Information:" }
```

Comment:
How is this relevant to the PR?

Reply:
Oops, that was not intentional, added it back.